.github/workflows/containers.yml

```diff
@@ -703,6 +703,121 @@ jobs:
+                volumes:
+                    - /$(pwd)/docker/nginx/load-balancer.conf:/etc/nginx
+                    - /$(pwd)/keystore/docker:/etc/ssl/private
+            api-catalog-services:
+                image: ghcr.io/balhar-jakub/api-catalog-services:${{ github.event.pull_request.number || 'latest' }}
+            caching-service:
+                image: ghcr.io/balhar-jakub/caching-service
+            discoverable-client:
+                image: ghcr.io/balhar-jakub/discoverable-client:${{ github.event.pull_request.number || 'latest' }}
+                env:
+                    APIML_SERVICE_CUSTOMMETADATA_APIML_LB_TYPE: authentication
+            discoverable-client-2:
+                image: ghcr.io/balhar-jakub/discoverable-client:${{ github.event.pull_request.number || 'latest' }}
+                env:
+                    APIML_SERVICE_HOSTNAME: discoverable-client-2
+                    APIML_SERVICE_CUSTOMMETADATA_APIML_LB_TYPE: authentication
+            mock-zosmf:
+                image: ghcr.io/balhar-jakub/mock-zosmf
+            discovery-service:
+                image: ghcr.io/balhar-jakub/discovery-service:${{ github.event.pull_request.number || 'latest' }}
+                env:
+                    APIML_DISCOVERY_ALLPEERSURLS: https://discovery-service:10011/eureka,https://discovery-service-2:10011/eureka
+            discovery-service-2:
+                image: ghcr.io/balhar-jakub/discovery-service:${{ github.event.pull_request.number || 'latest' }}
+                env:
+                    APIML_SERVICE_HOSTNAME: discovery-service-2
+                    APIML_DISCOVERY_ALLPEERSURLS: https://discovery-service-2:10011/eureka,https://discovery-service:10011/eureka
+            gateway-service:
+                image: ghcr.io/balhar-jakub/gateway-service:${{ github.event.pull_request.number || 'latest' }}
+                env:
+                    APIML_SERVICE_DISCOVERYSERVICEURLS: https://discovery-service:10011/eureka/,https://discovery-service-2:10011/eureka/
+                    APIML_ROUTING_INSTANCEIDHEADER: true
+                    APIML_LOADBALANCER_DISTRIBUTE: true
+            gateway-service-2:
+                image: ghcr.io/balhar-jakub/gateway-service:${{ github.event.pull_request.number || 'latest' }}
+                env:
+                    APIML_SERVICE_HOSTNAME: gateway-service-2
+                    SERVER_INTERNAL_PORT: 10027
+                    APIML_SERVICE_DISCOVERYSERVICEURLS: https://discovery-service:10011/eureka/,https://discovery-service-2:10011/eureka/
+                    APIML_ROUTING_INSTANCEIDHEADER: true
+                    APIML_LOADBALANCER_DISTRIBUTE: true
+        steps:
+            -   uses: actions/checkout@v2
+                with:
+                    ref: ${{ github.head_ref }}
+            -   name: Set up JDK 1.8
+                uses: actions/setup-java@v1
+                with:
+                    java-version: 1.8
+            -   name: Setup Docker
```

Comment:
I would remove this for standard runs. If anything then maybe we need to have a different base build image.

Reply:
It's already conditionally off by default